swift-sdk/Internal/RequestCreator.swift

```diff
@@ -546,7 +546,7 @@ struct RequestCreator {
 
         var body = [AnyHashable: Any]()
         
-        addUserKey(intoDict: &body)
+        setCurrentUser(inDict: &body)
 
```

Comment:
Seems weird that there is an impression request and a embedded session request. But, thats not dealing with this change.

Reply:
yes, I think it's legacy. seems good for a refactor stage later